Workaround for Windows VC++ 64-bit openssl compile.

rkongintel opened the pull request:
Just seeing if this compiles and works for other builds in CI. (Linux mbedtls/openssl). I've only tested Windows VC++ 2019 64-bit openssl/mbedtls (requester/responder tests).  

32-bit Windows build does not compile in main.

Signed-off-by: Kong, Richard <richard.kong@intel.com>

steven-bellock commented:
@jyao1 @rkongintel I fixed some typos and added a sentence on what the ultimate intent is. Please review.

Also, I feel like I tried something extraordinarily similar to this but couldn't get it to work. Good job for getting it to work!

jyao1 commented:
looks good

rkongintel commented:
> @jyao1 @rkongintel I fixed some typos and added a sentence on what the ultimate intent is. Please review.
> 
> Also, I feel like I tried something extraordinarily similar to this but couldn't get it to work. Good job for getting it to work!

Changes look good. @steven-bellock Thanks. It looks simple, but it was actually quite challenging.  I went through many experiments to figure out what was happening and to try different things to workaround it. Glad it is fixed and we can get the build enabled in main again.
